### ISSUE TITLE
Genesis v1.5.7 not uploading deps in pipeline correctly

### ISSUE THREAD

**Issue by bodymindarts** [starkandwayne/genesis#93]
Used latest genesis to update a bosh-release via a pipeline and ran into another regression.
Will debug it later on.
bosh should be update to 259 but 256.2 is being uploaded, and other releases are being downgraded :-S.
Output:
```
Deploying to BOSH director a5d90271-bb7e-4ffb-b14b-5807fe33f194

Target set to 'acf-global-proto-bosh'
Deployment set to '/tmp/build/028a4217/out/git/acf-global/alpha/manifests/.deploy.yml'
Acting as user 'admin' on 'acf-global-proto-bosh'
Checking whether release bosh/259 already exists...NO
Using remote release 'https://bosh.io/d/github.com/cloudfoundry/bosh?v=256.2'

Director task 3648
  Started downloading remote release > Downloading remote release. Done (00:00:20)

  Started verifying remote release > Verifying remote release. Done (00:00:00)

  Started extracting release > Extracting release. Done (00:00:01)

  Started verifying manifest > Verifying manifest. Done (00:00:00)

  Started resolving package dependencies > Resolving package dependencies. Done (00:00:00)

  Started processing 13 existing packages > Processing 13 existing packages. Done (00:00:00)

  Started processing 8 existing jobs > Processing 8 existing jobs. Done (00:00:00)

  Started release has been created > bosh/256.2. Done (00:00:00)

Task 3648 done

Started		2016-11-03 08:15:20 UTC
Finished	2016-11-03 08:15:41 UTC
Duration	00:00:21

Release uploaded
Acting as user 'admin' on 'acf-global-proto-bosh'
Checking whether release bosh-aws-cpi/60 already exists...YES
Acting as user 'admin' on 'acf-global-proto-bosh'
Checking whether release shield/6.3.6 already exists...YES
Acting as user 'admin' on deployment 'acf-global-alpha-bosh' on 'acf-global-proto-bosh'
Getting deployment properties from director...

Detecting deployment changes
----------------------------
resource_pools:
- name: bosh
  stemcell:
    sha1: 95ecc2709ac62f21d0a1c6cac023585c3919b825
    url: https://bosh.io/d/stemcells/bosh-aws-xen-hvm-ubuntu-trusty-go_agent?v=3263.8
releases:
- name: bosh
  version: '256.2'
  version: '259'
- name: bosh-aws-cpi
  sha1: 8e40a9ff892204007889037f094a1b0d23777058
  sha1: 3a5988bd2b6e951995fe030c75b07c5b922e2d59
  url: https://bosh.io/d/github.com/cloudfoundry-incubator/bosh-aws-cpi-release?v=60
  url: https://bosh.io/d/github.com/cloudfoundry-incubator/bosh-aws-cpi-release?v=53
- name: shield
  sha1: de6e56e8a10c7f07cc6464b1b5239562c4ce47cb
  sha1: 1918606e3fba5351470985e8d641a2ebe7e85912
  url: https://bosh.io/d/github.com/starkandwayne/shield-boshrelease?v=6.3.6
  url: https://bosh.io/d/github.com/starkandwayne/shield-boshrelease?v=6.2.1

Deploying
---------

Director task 3649
  Started preparing deployment > Preparing deployment. Failed: Release version 'bosh/259' doesn't exist (00:00:00)

Error 30006: Release version 'bosh/259' doesn't exist

Task 3649 error

For a more detailed error report, run: bosh task 3649 --debug
```

**Comment by jhunt** [comment]
I thought latest was 1.5.6?

**Comment by bodymindarts** [comment]
I released with the fix from #91 this morning to roll it out to client. It was working fine till we tried to update a release.

**Comment by bodymindarts** [comment]
Answer is to
```
rm global/releases/**/{url,sha1}
```
then things work as expected.
